librz/core/canalysis.c

```diff
@@ -2375,6 +2377,7 @@ static int core_analysis_graph_construct_nodes(RzCore *core, RzAnalysisFunction
 								free(body_b64);
 								free(title);
 								rz_diff_free(d);
+								rz_config_hold_free(hc);
```

Comment:
there is another `return` few lines above that does not have `rz_config_hold_free` either.